examples/reference/widgets/FileInput.ipynb

```diff
@@ -92,6 +128,8 @@
    "cell_type": "markdown",
    "metadata": {},
    "source": [
+    "## Restricting the File Types\n",
+    "\n",
```

Comment:
After uploading a `.png` file I run the `.clear` method. But it errors

![image](https://github.com/user-attachments/assets/9ae5741f-0340-4550-8e45-59f79804eb95)

```bash
ValueError: ClearInput event only applies to input models, i.e. instances of bokeh.models.widgets.InputWidget
```

Reply:
Fix by implementing new clear method.